### PR TITLE
Arbitrary pagination

### PR DESCRIPTION
**Pull request by okal**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-(defproject onaio/slickgrid-cljs "0.0.2-SNAPSHOT"
+(defproject onaio/slickgrid-cljs "0.0.2-arbitrary-page-counts-SNAPSHOT"
```
Comment (pld) [inline project.clj]: If it's going into master, this should probably be reverted

**Comment by pld** [comment]
From my limited context :+1:

File: project.clj (hunk 1 above)
Comment (okal) [inline project.clj]: I think that should only happen after QA.

**Comment by okal** [comment]
You can test it by running the branch `table-data-pagination` branch locally. Making sure it says "Showing 1 of `<actual number of pages based on submissions>` pages" rather than basing the number on the submissions held in memory.

File: project.clj (hunk 1 above)
Comment (pld) [inline project.clj]: Cool, as long as it's before merge

**Comment by okal** [comment]
This ended up being unnecessary. We re-implemented the paging component in Om.

**Comment by okal** [comment]
But it's good for Makefiles and stuff like that.

**Pull request merged**